apps/editors/tests/test_views.py

```diff
@@ -1288,15 +1292,12 @@ class TestUnlistedPendingQueue(TestPendingQueue):
 
     def setUp(self):
         super(TestUnlistedPendingQueue, self).setUp()
-        # These should be the only ones present in the queue.
-        self.expected_names = ['Pending One', 'Pending Two']
```

Comment:
It's a bit messy. As the _unlisted_ tests inherit from the listed tests, and we call the parent class (line 1294), we don't need to set `self.expected_names`. That's not harmful, but calling `self.expected_addons = self.get_expected_addons_by_names()` in the _unlisted_ tests again actually breaks the test `test_results_two_versions`because suddenly all versions from `generate_files()` now have two file attached, due to calling `Addon.with_unlisted.get_or_create()` in _test_models.py_. Took me a while to figure that out.